### PR TITLE
Add pause state to ps, touchup kpod man page

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Adds a pause state to kpod-ps and touches up the main kpod man page to list pause and unpause.

### CODE REVIEW

File: cmd/kpod/ps.go [hunk 1]
```diff
@@ -299,6 +299,8 @@ func getTemplateOutput(containers []*libkpod.ContainerData, opts psOptions) (psO
 			status = "Exited (" + strconv.FormatInt(int64(ctr.State.ExitCode), 10) + ") " + runningFor + " ago"
 		case runningState:
 			status = "Up " + runningFor + " ago"
+		case "paused":
```
Comment (rhatdan) [inline cmd/kpod/ps.go]: You have a constant defined above for "running", and "stopped" and "paused".  Need to be consistant.  I would define them all as constants.

Comment (rhatdan) [reply cmd/kpod/ps.go]: But better yet, include oci/oci.go and get the constants from there.

Comment (TomSweeneyRedHat) [reply cmd/kpod/ps.go]: Fixed

**Comment by TomSweeneyRedHat** [comment]
per @mrunalp, I'm not chasing test errors until #969 becomes stable/merged.

**Comment by mrunalp** [comment]
Needs rebate before testing again.

**Comment by mrunalp** [comment]
Known failure in e2e. this is good.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**